INSTALL_DOCKER.rst

```diff
@@ -32,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
```

Comment:
Ah, apologies @gorozco1 - my eyes didn't pick up the `#` - yes, good spot - will edit and re-PR. Thanks!